ibl_pipeline/acquisition.py

```diff
@@ -1,47 +1,13 @@
 import datajoint as dj
-
 from . import subject
 from . import reference, subject, action
+import os
 
-schema = dj.schema(dj.config.get('database.prefix', '') + 'ibl_acquisition')
-
-'''
-To simplify from Alyx schema, dropping file repositories as core
-parts of tables and moving to per-table blobs. A facility for
-tracking data provenance could still made avaialble via a table
-linked into individual model records as a non-primary attribute.
-
-This is probably not ideal w/r/t Alyx -
-
-one idea would be to merge use external storage with example
-field def being:
-
-  aligned_movie :  external  # motion-aligned movie
-
-and also have a separate 'phase 1' data schema, which is then coupled
-with phase 2+ for higher level data products.
-
-Also note: TimeScale items missing wrt TimeSeries items, which were:
-
-- PupilTracking.xyd
-- PupilTracking.movie
-- HeadTracking.xy_theta
-- HeadTracking.movie
-
-TimeScale not yet defined
-
-# SKIPPED:
-# <class 'data.models.DataRepositoryType'>
-# <class 'data.models.DataRepository'>
-# <class 'data.models.DatasetType'>
-# <class 'data.models.Dataset'>
-# <class 'data.models.FileRecord'>
-# <class 'data.models.DataCollection'>
-# <class 'data.models.Timescale'>
-# <class 'data.models.TimeSeries'>
-# <class 'data.models.EventSeries'>
-# <class 'data.models.IntervalSeries'>
-'''
+mode = os.environ.get('MODE')
+if mode == 'update':
```

Comment:
Actually, why is `update` treated specially here? Shouldn't the schema name be prefixed by `update_`?